features/step_definitions/fincap_pages_steps.rb

```diff
@@ -26,6 +26,30 @@
   )
 end
 
+Given(/^I have an article page layout setup with components$/) do
+  english_site = Comfy::Cms::Site.find_or_create_by(
```

Comment:
the set up of the site is already carried out at the top step `Given(/^I have an insight page layout$/) do`
Perhaps we can split that into it's own step, that we only call from the steps themselves? wdyt?

Reply:
Oh, I forgot about that line. Yes good idea.

Reply:
@willhall88 actually I discover with @margOnline a helper called `cms_site` to be used on cucumber steps so I just used ... and it worked! 🎩 Wdyt? ♻️

Reply:
😻